contracts/masset/Masset.sol

```diff
@@ -86,7 +95,9 @@ contract Masset is
         basketManager = IBasketManager(_basketManager);
 
         MAX_FEE = 2e16;
-        swapFee = 4e15;
+        swapFee = 6e14;
+        redemptionFee = 3e14;
+        cacheSize = 1e17;
```

Comment:
It might be good to test extreme values here: for instance, by running the test suite with 0, 1, and a value that corresponds to 100%.

Reply:
Will add.